symfony/plugins/orangehrmAttendancePlugin/Service/AttendanceService.php

```diff
@@ -85,12 +93,15 @@ public function checkForPunchInOverLappingRecords($punchInTime, $employeeId) {
     }
 
     /**
-     * get saved Attendance configuration
-     * @param $workflow, $state, $role, $action, $resultingState
-     * @return boolean
+     * @param $workflow
+     * @param $state
+     * @param $role
+     * @param $action
+     * @param $resultingState
+     * @return bool
      */
-    public function getSavedConfiguration($workflow, $state, $role, $action, $resultingState) {
-
+    public function getSavedConfiguration($workflow, $state, $role, $action, $resultingState): bool
```

Comment:
we can remove this method, since it only wraps dao method call

Reply:
done